docs/v2/guides/posts.md

```diff
@@ -283,6 +283,88 @@ If you’re used to using `get_posts()` instead of `WP_Query`, you will have to
 
 Of course, the other main difference is that instead of returning plain `WP_Post` objects, `Timber::get_posts()` returns instances of `Timber\Post`.
 
+## Serialization
+
+When you want to work with post data in JavaScript, you will want to convert it to JSON first.
+
+Under normal circumstances, this wouldn't be a problem. However, Timber posts are instantiated lazily. This means that most methods will only calculate and return a value the first time you call them. Take the `Timber\Post::link()` method for example. You use it to get the permalink for a post.
```

Comment:
Before this said: `Under normal circumstances, this would be a problem. ` — but I _think_ you meant the opposite

Reply:
> but I _think_ you meant the opposite

Yep 😊👍